packages/flutter_hooks/lib/src/animation.dart

```diff
@@ -179,6 +180,7 @@ class _TickerProviderHookState
 
   @override
   void dispose() {
+    super.dispose();
```

Comment:
Needs to be at the end of the dispose method

Reply:
This was resolved but still needs to be changed :)

Reply:
Oups i was thinking it was done by your suggested changes, I'll do that now

Reply:
Done now :)